teuthology/suite/placeholder.py

```diff
@@ -57,6 +57,13 @@ def _substitute(input_dict, values_dict):
         'admin_socket': {
             'branch': Placeholder('ceph_branch'),
         },
+        'ceph_ansible': {
```

Comment:
Yeah, I think they should go there. Placeholders aren't a great place for defaults - despite the fact that we have some there.